### PR TITLE
Microsoft Teams enhancements

### PR DESCRIPTION
**Pull request by Shellyber**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/19312

## Description
Adding 2 new commands:
* add-user-to-channel
* create-channel

## Required version of Demisto
5.0.0

## Does it break backward compatibility?
   - No

## Must have
- [ ] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

## Technical writer review
Mention and link to the files that require a technical writer review.
- [ ] [YAML file](link)
- [ ] [CHANGELOG](link)

### CODE REVIEW

**Comment by Shellyber** [comment]
The remove user from private channel doesn't seem to work in the beta api. We will try to contact MS about it. 
Will open a new enhancement for this command.

**Comment by Shellyber** [comment]
@kirbles19 Please review the CHANGELOG.md file and also the YML file for 2 new commands:
* create-channel
* add-user-to-channel

File: Integrations/MicrosoftTeams/MicrosoftTeams.py [hunk 1]
```diff
@@ -712,6 +712,100 @@ def get_users() -> list:
...
+        if member in {user.get('displayName', ''), user.get('mail'), user.get('userPrincipalName')}:
+            found_member = True
+            user_id = user.get('id', '')
+            break
+    if not found_member:
+        demisto.results({
+            'Type': entryTypes['warning'],
+            'Contents': f'User {member} was not found',
+            'ContentsFormat': formats['text']
+        })
```
Comment (ShahafBenYakir) [inline Integrations/MicrosoftTeams/MicrosoftTeams.py]: Is this intentionally a warning and not an error? 
I think we should also check if `user_id` exists. Like this
`if not found_member or not iser_id:`

File: Integrations/MicrosoftTeams/MicrosoftTeams.py [hunk 2]
```diff
@@ -814,6 +908,17 @@ def create_channel(team_aad_id: str, channel_name: str, channel_description: str
...
 
+def create_channel_command():
+    channel_name: str = demisto.args().get('channel_name', '')
+    channel_description: str = demisto.args().get('description', '')
+    team_name: str = demisto.args().get('team', '')
+    team_aad_id = get_team_aad_id(team_name)
+
+    channel_id: str = create_channel(team_aad_id, channel_name, channel_description)
+    if channel_id:
+        demisto.results(f'The channel "{channel_name}" was created successfully')
```
Comment (ShahafBenYakir) [inline Integrations/MicrosoftTeams/MicrosoftTeams.py]: Why not also add the `channel_id` to context? I think this could help our users in their next tasks

**Comment by ShahafBenYakir** [comment]
Good job

**Comment by kirbles19** [comment]
@Shellyber Done.

**Pull request merged**